### ISSUE TITLE
Move movies even if they are not on the list

### ISSUE THREAD

**Issue by jpborjas** [Radarr/Radarr#455]
Enhancement Request. When I added a movie to SABnzbd for manual download, Radarr never picked it up when finished, and I was told that it only looks for movies that it is looking for, that it knows of. If I need to do that, it adds an extra step that is really unnecessary. Couch Potato does it like a service, movie I see in there, I'm renaming and moving over. This is more aggravating when I learned that I needed to use the Drone Factory Folder for that, but when I tried it, it has to be a folder other that the "from" folder for Radarr. 

Right now I have Couch Potato on, just to move over my "one offs".

**Comment by jpborjas** [comment]
I added this to FeatHub

**Comment by benbar21** [comment]
You can already do this with the drone factory. All you need to do is have two different categories/folders for movies.

For example, you could have two categories: movies and movies_auto. Movies would go to your drone factory folder (let's pretend its at m:/downloads/movies), and then Radarr would find them when it scans the folder. Movies_auto would go to some folder that is not part of the drone factories (maybe m:/downloads/movies_auto). 

Whenever radarr downloads a movie it'd send it to Sab with the movies_auto tag. Radarr would then automatically know that the movie will end up in .../movies_auto. Whenever you manually download a movie you'd use the movies category. 

Further, there is the 'manual import' option under wanted. While this isn't automatic, it also will import movies that you downloaded manually. 

I have no idea if either of these features are currently working as Radarr is still very new, but it should, if not now, eventually do what you want it to do.

**Comment by jpborjas** [comment]
Bebar, I'm going to give your example a try, but ultimately, the way Couch Potato does it, where everything in the monitored folder (drone) get's renamed and moved, I think is the easiest way. I opened a case yesterday and it turns out I wasn't understanding how and why there were two options for the rename folder, the Drone, and the Mapped. In any case, simpler is better, and simpler code too. Also, the movie doesn't need to be in Radarr for it to rename it and move it. If a friend gives me a movie on a thumb drive, I would like just drop it in the "from" folder, and Radarr should rename it, move it, delete it from the "from" folder, and add it to my list. Simple.

**Comment by benbar21** [comment]
@jpborjas 

So Couchpotato is meant to be set up this way too, the documentation just isn't clear. My 'from' folder in CP is different from the folder that NZBGet downloads to. The purpose of the renamer for Couchpotato is to be able to rename movies that you downloaded outside of CP.

Sure, in CP you can set the two folders to be the same, but then at times they conflict with each other: the renamer and downloader will both try to rename a movie, which causes errors. To be honest couch potato has worked for me much more consistently once I switched to using two different folders.

There really is practically speaking no difference between what I am saying you should do and what you want to do. You can have a folder (perhaps m:/downloads/movies) that the drone factory is set to. If you download a movie, or a friend gives you a movie on a thumb drive, you can just put that movie in m:/downloads/movies. The drone factory will then pick it up.

Unless I'm mistaken @onedr0p won't ever make it so that the drone factory and nzb download location can be the same because because then the two will conflict with each other and cause errors.

so tldr; you're actually misusing Couchpotato, but the proper way of doing things is practically speaking identical to what you want to do save for the initial set up.

**Comment by jpborjas** [comment]
So, if your from folder in CP is different than your download folder from NZBGet downloads, how does CP get the movies automatically, or do you have to move them manually? I've never had an issue with files getting renamed. Here's my setting:

In SABnzbd, my temporary download folder is: /Users/myuser/Downloads/Incomplete
Completed Download Folder is: /Users/myuser/Movies
My Category Folder Path for Movies is: /Users/myuser/Movies/Movies      (I separate Movies from TV Shows)

When SAB is done downloading, then it creates a folder with the name of the movie and adds the word _UNPACK_ before the name. Maybe that is what keeps CP from start renaming. My CP is set to run the renamer every 1 minute. I'm watching it download and rename "Furious 7" right now, no issues. It also removes the empty folder once it is done. 

Another thing I noticed is that I have Sonarr set up using the Drone Factory and not the Remote Path Mappings, and it works great. Everything is pointing back at Radarr's Drone not working properly.

**Comment by benbar21** [comment]
Both the NZBGet and SAB APIs announce to CP/Radarr/Sonarr/whatever where the file was downloaded. Try it out: set sab to download to a random folder separate from your 'from' folder and it'll still work! 

In CP my 'from' folder is m:/downloads/movies
my category is movies_auto
and the folder for movies_auto is m:/downloads/auto/movies

Adding unpack may be what's preventing CP from breaking, since the renamer likely isn't recognizing a folder like "UNPACK The Matrix Unloaded 1080p release"

In terms of Radarr specifically the drone factory may not be working right now since lots of features aren't. I'll give it a try later tonight and let you know if it works for me. 

But, in general, you should keep the 'from' folder or drone folder or whatever separate from the folder that NZBGET/Sab download to to prevent conflicts.

**Comment by jpborjas** [comment]
I may be misusing SAB. I don't even know where to configure it so it tells CP/Radarr/Sonarr when it's done, but I don't know if I need to. Don't want to brake CP and Sonarr. 

The _UNPACK_ part is automatic, I didn't set that up. Let me know about your tests. 

I tried to install the Continuous release for Radarr, but it stayed on 0.2.0.226. Drone was supposed to be fixed on #96.

**Comment by benbar21** [comment]
It's all automatic! CP gets it automatically just by sending the download to SAB.

I'll let ya know later tonight! I haven't followed the Radarr releases that much yet, tbh
